### ISSUE TITLE
Login popup does not close due to the language of the operating system.

### ISSUE THREAD

**Issue by AlejandroBlackburn** [shaikhsajid1111/facebook_page_scraper#80]
The login popup with the `[aria-label="Close"]` does not close when you have a language different from English in your operating system. It does not matter if the browser is in English, the popup does not close.

I have a small proposed fix in the `__close_modern_layout_signup_modal` method and wanted to ask if a pull request can be made.

Additionally I wanted to know if you can add support for passing the driver location as an argument in the constructor. At the moment, Webdriver Manager is buggy with the new version of Chrome and cannot be used. I have a small fix for this bug

**Comment by shaikhsajid1111** [comment]
Yes, this Library is open to changes which can improve it. Please feel free to raise a PR @AlejandroBlackburn
